.coveragerc

```diff
@@ -2,4 +2,4 @@
 relative_files = True
 
 # Use 'source' instead of 'omit' in order to ignore 'tests/unit/__init__.py'
-source = hello_world
+source = floqast_sftp
```

Comment:
hmm, can't seem to see line coverage in coveralls. it says `source not available` when click on the source files.

Reply:
I can see it here for an out-of-date build: https://coveralls.io/builds/74889781
but it took a couple refreshes to get past an ajax error so maybe coveralls is having some sort of partial outage